docs/platform/what-is-tizen/overview.md

```diff
@@ -30,22 +30,25 @@ From Tizen 3.0, all profiles are built on top of a common, shared infrastructure
 
 ![Common infrastructure for various profiles](media/about_tizen_2.png)
 
-## IoT Capability
+## Multi Device Experience
 
-The most important strengths that enable Tizen to move to the next level are IoT and convergence capabilities:
+Users usually use multiple smart devices in various situations. Tizen enables not only connecting those devices but also leveraging every device's capabilities for providing differentiated experience to the users.
 
-- **Device Convergence**: Tizen provides a device-to-device convergence framework for easy data sharing and direct remote control between devices. Using this framework, you can handle events or data on the remote device as if they were local. Device Convergence in Tizen opens up a host of capabilities that developers can use to create a new way of experiencing connectivity.
+- **Supporting latest connectivity standards**: Tizen supports novel connectivity standards. The latest Wi-Fi standards like Wi-Fi Aware, Wi-Fi Easy Connect are supported by the platform and it adds support for new standards continuously. Matter and Thread protocols are the next generation IoT protocol for smart home devices. Especially, Tizen will support Matter protocol from the first release (1.0) of the open source matter SDK.
 
-  **Figure: Device Convergence**
+  **Figure: Connectivity Standards**
+  
+  <img src="media/about_tizen_7.png" alt="Connectivity Standards" width="600"/>
+  
+- **Services leveraging multiple devices' capabilities**: Tizen provides easy ways of implementing services which work on multiple devices. For services on multiple devices, Tizen provides methods for discovering services among connected devices and communicating each other in efficient way. It is not bounded to Tizen-powered devices. Cross platform APIs are available for Tizen-Android inter-working.
```

Comment:
I have added a preposition and an article to make it better understandable. Let me know if this seems correct to you. 
```suggestion
- **Services leveraging multiple devices' capabilities**: Tizen provides easy ways of implementing services which work on multiple devices. For services on multiple devices, Tizen provides methods for discovering services among connected devices and communicating with each other in an efficient way. It is not bounded to Tizen-powered devices. Cross platform APIs are available for Tizen-Android inter-working.
```